index.html

```diff
@@ -1,527 +1,729 @@
+                      />
+                    </svg>
+                  </button>
                 </div>
-            </div>
-        </section>
+              </div>
+            </li>
+            <li>
+              <div class="main-frame">
+                <div class="frame-header">
+                  <div class="header-txt">
+                    <p>Skills</p>
+                  </div>
+                  <button class="normal-btn">
+                    <svg
+                      width="24"
+                      height="25"
+                      viewBox="0 0 24 25"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                        fill="#505F79"
+                      />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact">
+        <div class="content">
+          <div class="txt-holder">
+            <h2 class="primary-txt">Contact me</h2>
+            <p class="detail-txt">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+
+          <div class="contact-form">
+            <form action="https://formspree.io/f/xyyoqwwb" method="post">
+              <ul>
+                <li>
+                  <input
+                    type="text"
+                    placeholder="Write your name here"
+                    name="user_name"
+                    id="name"
+                    required
+                    maxlength="30"
+                  />
+                </li>
+                <li>
+                  <input
+                    type="email"
+                    placeholder="Write your email here"
+                    name="user_email"
+                    id="mail"
+                    required
+                  />
+                </li>
+                <li>
+                  <textarea
+                    placeholder="Write your message here"
+                    name="user_message"
+                    id="msg"
+                    required
+                    maxlength="500"
+                  >
+                  </textarea>
+                </li>
+                <li>
+                  <button class="submit-btn" type="submit">Get in Touch</button>
```

Comment:
- [ ] You can consider aligning the button to the center  of the form and reducing the form width so it looks like the provided Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220708_054238](https://user-images.githubusercontent.com/25227022/177918586-70fe6d72-fcb9-47f5-ada6-63704bd60848.png)|![Screenshot_20220708_054253](https://user-images.githubusercontent.com/25227022/177918598-6f1e1e9a-2e39-4a76-ba13-cce103c0fef9.png)|